src/utils/componentService.js

```diff
@@ -0,0 +1,28 @@
+import api from '../config/api'
+
+const createQuery = (target = '', query) => `
+query {
+  repository(owner: "Wolox", name: "frontend-cookbook") {
+    content: object(expression: "components${target}") {
+      ${query}
+    }
+  }
+}`
+
+const componentFileQuery = (componentType, component, filename) =>
+  createQuery(`:${componentType}/${component}/${filename}`, `
+    ... on Blob {
+      text
+    }`
+  )
+
+export async function getComponentFiles(componentType, component) {
+  const [htmlResponse, scssResponse] = await Promise.all([
+    api.post('', { query: componentFileQuery(componentType, component, 'index.html') }),
```

Comment:
Is there any way to get both files in one request?

Reply:
This is the best way since you need to specify the target in the query, like:
```
content: object(expression: "components${target}") {
```
being the target each file